Header header
uint32 stamp
float32 lat
float32 lon
float32 height
float32 x
float32 y
float32 z
float32 vel_x
float32 vel_y
float32 vel_z
float32 ground_speed
float32 head_motion
int8 fix_type